app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SessionsController < Devise::SessionsController
+  include Devise::Controllers::Rememberable
+  def create
+    delete_legacy_cookie
+    super
+  end
+
+  def destroy
+    delete_legacy_cookie
+    cookies.delete :remember_user_token, domain: ".#{SiteConfig.app_domain}"
+    # Let's say goodbye to all the cookies when someone signs out.
+    cookies.clear(domain: cookie_domain)
+
+    super
+  end
+
+  private
+
+  def legacy_cookie_domain
+    Rails.env.production? ? SiteConfig.app_domain : nil
+  end
+
+  def cookie_domain
```

Comment:
Did we intentionally add this back in? This has been removed on `master`: https://github.com/forem/forem/pull/10211/files#diff-d5241d488259f32ecbe2f636133e5ddaL23

Reply:
Yeah, I meant to pull latest, I guess I didn't properly resolve things. Good call.